core/io/http_client.cpp

```diff
@@ -578,6 +584,7 @@ void HTTPClient::_bind_methods() {
 
 	ObjectTypeDB::bind_method(_MD("connect:Error","host","port","use_ssl","verify_host"),&HTTPClient::connect,DEFVAL(false),DEFVAL(true));
 	ObjectTypeDB::bind_method(_MD("set_connection","connection:StreamPeer"),&HTTPClient::set_connection);
+	ObjectTypeDB::bind_method(_MD("get_connection"),&HTTPClient::get_connection);
```

Comment:
Could you add ``:StreamPeer`` after the ``get_connection`` so that the doctool knows about the return type? (see the argument for ``set_connection`` above).